plugins/out_http/http.c

```diff
@@ -113,6 +113,9 @@ static int http_post(struct flb_out_http *ctx,
                         ctx->host, ctx->port,
                         ctx->proxy, 0);
 
+
+    flb_debug("[http_client] proxy host: %s port: %i", c->proxy.host, c->proxy.port);
```

Comment:
since this debug message is inside a plugin code, it should use flb_plg_debug(ctx->ins, "..."), on this case don't need the component prefix since the API will put it there automatically

Reply:
Fixed.